docs/ref/clvm.md

```diff
@@ -666,14 +667,16 @@ Next we'll show the cost of each CLVM operator, as well as the cost of the outpu
 | `pubkey_for_exp`    | 1325730   | -            | 38            |
 |                     |           |              |               |
 
-Finally, three of CLVM's conditions also have an associated cost:
+Next, three of CLVM's conditions also have an associated cost:
 
 | operator            | cost    |
 | ------------------- | ------- |
 | `CREATE_COIN`       | 1800000 |
 | `AGG_SIG_UNSAFE`    | 1200000 |
 | `AGG_SIG_ME`        | 1200000 |
 
+Finally, each byte of data that gets added to the blockchain has a cost of 12,000. Spendbundles are created using serialized bytecode, calculated by running [opc](https://chialisp.com/docs/debugging#opd-and-opc) on the original CLVM code. Each two-character pair of this serialized bytecode translates to one byte on the blockchain, with a cost of 12,000.
```

Comment:
> Each two-character pair of this serialized bytecode translates to one byte on the blockchain, with a cost of 12,000.

I think this is an odd description. Bytes are often printed as hex to be human readable. That doesn't really affect that serialization format. I don't think we need to have a name for the serialized format printed as hex; "serialized bytecode" should just refer to the bytes. This sentence makes it sound like the "serialized bytecode" refers to the hexadecimal representation, which I think is misleading.

Another nit-pick is that "bytecode" often refers to programs in some machine code-like instruction set (like, compiled python or java code). In our case, it's really just a serialization of a tree. That (clvm) tree represents both data and instructions mixed together. I think we could just refer to these bytes as our serialization format of chialisp (or clvm) programs.